desktop/ui/src/main/java/org/datacleaner/widgets/properties/SingleFilePropertyWidget.java

```diff
@@ -94,9 +95,9 @@ public SingleFilePropertyWidget(ConfiguredPropertyDescriptor propertyDescriptor,
             _filenameField.setFile(currentValue);
         }
 
-        _filenameField.addFileSelectionListener(new FileSelectionListener() {
+        _filenameField.addSelectionListener(new FileSelectionListener() {
```

Comment:
This method name change will break compatibility I guess. And now that the interface was reverted to name "FileSelectionListener" then I guess also the old method name makes sense.